pkg/controller/volume/attachdetach/cache/desired_state_of_world.go

```diff
@@ -195,6 +195,8 @@ func (dsw *desiredStateOfWorld) AddPod(
 
 	nodeObj, nodeExists := dsw.nodesManaged[nodeName]
 	if !nodeExists {
+		// TODO: Why not auto-create the node?
+
```

Comment:
I cannot think of a strong reason to against it. I think the logic is just to make sure there is that node is added before adding pods